Directory.Build.props

```diff
@@ -74,7 +74,7 @@
 
     <!-- Indicates this is not an officially supported release. Release branches should set this to false. -->
     <!-- Keep it in sync with PRERELEASE in eng/native/configureplatform.cmake -->
-    <IsPrerelease>true</IsPrerelease>
+    <IsPrerelease>false</IsPrerelease>
```

Comment:
This should be set to `true` on non release branches. It seems like you have extra commits?

Reply:
I think i started from the last prerelease branch, which included that commit.
It should be easily reversible

Reply:
Yeah, we should put your commits on top of a branch created from master to make it more reviewable.

Reply:
I agree with Santi, in general your changes look reasonable and valuable for certain scenarios, we just need to figure out the necessary logistics. I would be also grateful if @janvorli could take a look at the cmake changes as he's one of our biggest linux / cmake experts.